src/Apps/Collect/Components/Base/CollectRefetch.tsx

```diff
@@ -1,18 +1,16 @@
-import { CollectArtworkFilterRefetch_viewer } from "__generated__/CollectArtworkFilterRefetch_viewer.graphql"
+// import { CollectRefetch_viewer } from "__generated__/CollectRefetch_viewer.graphql"
 import { isEqual } from "lodash"
 import React, { Component } from "react"
 import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
 import { Responsive } from "Utils/Responsive"
 import { CollectArtworkGridRefreshContainer as ArtworkFilter } from "./CollectArtworkGrid"
-interface CollectArtworkFilterRefetchProps {
+interface CollectRefetchProps {
   filters: any
-  viewer: CollectArtworkFilterRefetch_viewer
+  viewer: any
```

Comment:
Is there a reason to type this as any? (E.g. limitation in our type emitting code?)

Reply:
I had to set it to `any` while I was renaming modules but I'll add the type back now 👍